content/configuration/dev-server.md

```diff
@@ -252,6 +337,29 @@ https: {
 
 This object is passed straight to Node.js HTTPS module, so see the [HTTPS documentation](https://nodejs.org/api/https.html) for more information.
 
+Usage via the CLI
+
+```bash
+webpack-dev-server --https
+```
+
+To pass your own certificate via the CLI use the following options
+
+```bash
+webpack-dev-server --https --key /path/to/server.key --cert /path/to/server.crt --cacert /path/to/ca.pem
+```
+
+
+## `devServer.info` - CLI only
+
+`boolean`
+
+Output cli information. It is enabled by default.
+
+```bash
+webpack-dev-server --info=false
+```
+
 
 ## `devServer.inline` - CLI only
```

Comment:
No worries, thanks again for this update! If you don't mind I may ping you in the future if there's other questions re this page since it seems like you have some familiarity with the webpack-dev-server repo.